webpack/utils/useKatelloDocUrl.js

```diff
@@ -0,0 +1,18 @@
+import URI from 'urijs';
+import { useForemanVersion } from 'foremanReact/Root/Context/ForemanContext';
+import { foremanUrl } from 'foremanReact/common/helpers';
+
+// useKatelloDocUrl('Managing_Content, '#Products_and_Repositories_content-management') =>
+// https://docs.theforeman.org/3.7/Managing_Content/index-katello.html#Products_and_Repositories_content-management
+export const useKatelloDocUrl = (guide = 'Managing_Content', hash = '') => {
+  // in dev you'll have to replace the Foreman version in the url with the latest published one
+  const foremanVersion = useForemanVersion();
+  const rootUrl = `https://docs.theforeman.org/${foremanVersion}/`;
+
+  const section = `${guide}/index-katello.html${hash}`;
```

Comment:
This doesn't belong in the frontend. By having it in the LinksController we can apply branding, which is crucial for Red Hat to properly supports its customers. Note that Red Hat also uses a different file name and directory structure.